### PR TITLE
2 1 tempo

### PR DESCRIPTION
**Pull request by YannYves**
form start

### CODE REVIEW

File: src/components/form/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  };
+
+  // Handle fields change
+  const handleChange = (e, input) => {
+    e.preventDefault();
+    setData({ ...data, [input]: e.target.value });
+  };
+
+  const handleChangeTest = (e, input) => {
+    // e.preventDefault();
```
Comment (LoicBrassart) [inline src/components/form/Form.jsx:39]: Dead code

File: src/components/form/Form.jsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    setData({ ...data, [input]: e.target.value });
+  };
+
+  const handleSubmit = (e) => {
+    // console.log('Un essai a été envoyé : ' + data.value);
```
Comment (LoicBrassart) [inline src/components/form/Form.jsx]: Dead code

File: src/components/form/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    e.preventDefault();
+  };
+
+  // const { step } = step;
```
Comment (LoicBrassart) [inline src/components/form/Form.jsx]: Dead code

File: src/components/form/Question3a.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        </div>
+        <form>
+          <div className="row">
+            {/* <input
+              className="formCard col-10 col-sm-6 py-4 px-0 px-3 px-lg-5 mb-5 mx-auto d-flex justify-content-between"
+              placeholder="Montant des travaux"
+              // type="text"
+              onChange={handleChange('question3a')}
+              defaultValue={values.question3a}
+            /> */}
```
Comment (LoicBrassart) [inline src/components/form/Question3a.jsx]: Dead code

**Review by JulienMartos — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**